templates/Deepcell_Postprocessing.ipynb

```diff
@@ -14,12 +14,16 @@
    "outputs": [],
```

Comment:
<p>Again, I think the imshow's can just go inside the <code>visualize_segmentation</code>  function right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/193/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>